src/components/FormInput/index.tsx

```diff
@@ -22,6 +22,7 @@ interface Props {
   touched?: boolean;
   submitCount?: number;
   inputRef?: any;
+  defaultValue?: string | number | readonly string[] | undefined;
```

Comment:
You don't need to explicitly use the `undefined` type here because you're already stating it could be undefined when using the optional `?` operator.
You can simply go like: 
```
  defaultValue?: string | number | readonly string[];
```